omnibus/config/software/workup.rb

```diff
@@ -57,7 +57,8 @@
 build do
   env = with_standard_compiler_flags(with_embedded_path)
 
-  gem 'install pkg/workup-0.1.6.gem', env: env
+  gem 'build workup.gemspec', env: env
```

Comment:
It may not. The clobber tasks knows about the pkg directory, I dont think it would remove .gems in the root of the repo